From 1cf032eaad80ae48c64b3760f07747d920c0b384 Mon Sep 17 00:00:00 2001
From: Rob Clark <robdclark@gmail.com>
Date: Sat, 17 Jan 2015 09:53:15 -0500
Subject: [PATCH 04/41] drm/msm: change kconfig for msm-3.18 kernel

- use ARCH_MSM instead of ARCH_QCOM
- remove dependency on Common Clock Framework

Change-Id: I4f18bac4427de498e5c0adff14c176f6d614abc5
Signed-off-by: Stephane Viau <sviau@codeaurora.org>
---
 drivers/gpu/drm/msm/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/msm/Kconfig b/drivers/gpu/drm/msm/Kconfig
index 08ba8d0..bf4d2db 100644
--- a/drivers/gpu/drm/msm/Kconfig
+++ b/drivers/gpu/drm/msm/Kconfig
@@ -2,8 +2,8 @@
 config DRM_MSM
 	tristate "MSM DRM"
 	depends on DRM
-	depends on ARCH_QCOM || (ARM && COMPILE_TEST)
-	depends on OF && COMMON_CLK
+	depends on ARCH_MSM || (ARM && COMPILE_TEST)
+	depends on OF
 	select REGULATOR
 	select DRM_KMS_HELPER
 	select DRM_PANEL
-- 
2.8.2

